packages/SwingSet/test/test-vpid-liveslots.js

```diff
@@ -652,26 +664,38 @@ async function doVatResolveCase4(t, mode) {
   await endOfCrank();
 
   const expectedP4 = nextP();
+  const expectedP5 = nextP();
+  let expectedArg = expectedP4;
```

Comment:
name this `expectedThreeArg`, to make it clear which method it's associated with. ditto for `expectedThreeResult`